composer.json

```diff
@@ -20,7 +20,7 @@
     ],
 
     "require": {
-        "php": "7.0 - 7.1",
+        "php": "7.0 - 7.2 ||7.2.*@dev",
```

Comment:
Doesn't the `--ignore-platform-reqs` option of Composer allows to get around that?